obspy/core/tests/test_event.py

```diff
@@ -7,13 +7,16 @@
 from obspy.core.event import readEvents, Catalog, Event, WaveformStreamID, \
     Origin, CreationInfo, ResourceIdentifier, Comment, Pick
 from obspy.core.utcdatetime import UTCDateTime
-from obspy.core.util.testing import ImageComparison, HAS_COMPARE_IMAGE
+from obspy.core.util.base import getBasemapVersion
+from obspy.core.util.testing import ImageComparison
 from obspy.core.util.decorator import skipIf
 import os
 import sys
 import unittest
 import warnings
 
+BASEMAP_VERSION = getBasemapVersion()
```

Comment:
Huh, shouldn't more newlines be required around this? Also, it should go after the below import. In fact, it could replace that import, couldn't it?